dashboard/v1.1/src/pages/Input/components/MaterialLabelSelector.tsx

```diff
@@ -61,12 +63,13 @@ export default function MaterialLabelSelector(props: LabelSelectorType) {
         return labels;
       })
       .then(labels => {
+        if(labelList.length===0)
```

Comment:
Might as well make this piece informative by either adding curly braces or comment on top.